Automattic/wp-calypso
# Staging Sites: Hide staging sites module on Pro Plan

* Closes https://github.com/Automattic/dotcom-forge/issues/2019

## Proposed Changes

* This PR hides the staging sites card on the sites that have Pro Plan. 

## Testing Instructions

* Make sure you have a WordPress.com site with a Pro Plan (note: it is a legacy plan, you would need to add it to a site from Store Admin)
* Instal any plugin on that site to make it Atomic
* Navigate to `Settings > Hosting Configuration` : /hosting-config/{site}
* Check that the **Staging Sites** module is not present:

<img width="1211" alt="Screenshot 2023-03-30 at 3 16 24 PM" src="https://user-images.githubusercontent.com/25575134/228848156-10fba0cd-90d5-4aa4-bc11-63f94cdb4230.png">

* Go to an Atomic site with a Business plan
* Navigate to `Settings > Hosting Configuration` : /hosting-config/{site}
* Check that the Staging Site module is present:

<img width="1080" alt="Screenshot 2023-03-30 at 3 18 59 PM" src="https://user-images.githubusercontent.com/25575134/228848909-55eb2359-6671-4462-bf32-8202e8a8e5b4.png">


## Pre-merge Checklist
